docs/release-notes/release-notes-v1.3.0.md

```diff
@@ -0,0 +1,89 @@
+# Pyserini Release Notes (v1.3.0)
+
++ **Release date:** November 1, 2025
++ **Anserini dependency:** v1.3.0
++ **Lucene dependency:** v9.9.1
+
+## Summary of Changes
+
++ Added 2CR for M-BEIR and BRIGHT (prebuilt indexes, bindings, etc.).
```

Comment:
query-side bm25 is also between July 16 and Nov 1:
https://github.com/castorini/pyserini/pull/2240